cmd/commands/server/command_server.go

```diff
@@ -34,11 +34,17 @@ type Command struct {
 	vpnServerFactory func(sessionManager session.Manager, serviceLocation dto_discovery.Location,
 		providerID identity.Identity, callback state.Callback) *openvpn.Server
 
-	vpnServer *openvpn.Server
+	vpnServer          *openvpn.Server
+	openvpnBinaryCheck func() error
```

Comment:
Why not `checkOpenvpn` as in client command?

Reply:
Saw this after PR was submitted. Will make the names the same